### DISCUSSION TITLE
Get Branch name in workflow_dispatch workflow

### DISCUSSION THREAD

**Discussion by giet12aei033** [community/community#124211] [Actions]
### Select Topic Area

General

### Body

<img width="331" alt="image" src="https://github.com/community/community/assets/166718827/8aca22e5-b54f-43a1-9291-ea838a4fef66">

I am using workflow_dispatch to trigger my workflow manually. I am selecting branch from drop down menu, in order to test i need to select branch other than main. How can i get the selected value in my env dynamically when i select branch

> jobs:
    liquibase_Steps:
        uses: xyz.github/workflows/reusable-workflow.yaml@main
        with:
            K8S_NAMESPACE: ${{ inputs.K8S_NAMESPACE }}
            BUILD_GIT_URL: 'git@github.com:xyz/xyz.git'
            GIT_BRANCH: ${{ github.ref_name }}
            
is above correct way yo reference it?

**Comment by msanft** [comment]
Yes, you can use `${{ github.ref_name }}`, or `${{ github.ref }}`, which is the full ref (something like `refs/tags/..` for example).

More information can be found in the [relevant docs](https://docs.github.com/en/actions/learn-github-actions/contexts#github-context).

**Reply by giet12aei033** [reply]
Thankyou
